.circleci/base_config.yml

```diff
@@ -42,7 +51,7 @@ parameters:
   # these are basically global constants
   build-targets:
     type: string
-    default: "arangod arangoimport arangoexport arangodump arangorestore arangobench frontend"
+    default: "arangod arangoimport arangoexport arangodump arangorestore arangobench frontend arangovpack"
```

Comment:
Why do we need `arangovpack` here?

Reply:
RTA expects to find all binaries that are there in distribution packages. `arangovpack` is part of that. We probably should have tests for it?